### PR TITLE
Use `node:` specifier for crypto import

### PR DESCRIPTION
**Pull request by alexgleason**
Improves Deno support.

So, I see you're overhauling this library in #92, but I figured I'd submit this anyway so you can see the diff.

Importing `node:crypto` is supported in both [Deno](https://deno.land/manual@v1.30.3/node/node_specifiers) and [Node.js](https://nodejs.org/api/esm.html#node-imports) as far back as v14.18.0 (the oldest supported LTS). And it simplifies things for Deno users a lot.

In #92 you could still `require('node:crypto')` to get the same advantages. Although... I think Deno might take the web APIs in that case anyway(?)

### CODE REVIEW

**Comment by paulmillr** [comment]
Great! v2.0 is behind the corner -- however, I could reuse this in noble-crypto / hashes. Will need to test deno with them.

**Comment by paulmillr** [comment]
Not great — big issues in hashes/curves with them. Seems like v2 will be the way to go for deno.

We can't use `import` because v2 will be able to run natively in browsers. Which can't import `node:crypto`.

**Comment by paulmillr** [comment]
tl;dr: v2 will use webcrypto, which is present in deno

**Comment by alexgleason** [comment]
Cool, I think that's a good solution. Thanks for taking a look!

**Comment by paulmillr** [comment]
Also did you know deno now natively supports npm modules? Google: npm specifier deno
